### PR TITLE
Pipiot

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/pipiot/peopleSense/uplink.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+function consume(event) {
+  const payload = event.data.payloadHex;
+  const bits = Bits.hexToBits(payload);
+  let topic = "default";
+  const data = {};
+  const lifecycle = {};
+
+  const cd = Bits.bitsToUnsigned(bits.substr(0, 1));
```
Comment (mariusgiger) [inline types/pipiot/peopleSense/uplink.js:36]: what is cd?

File: types/pipiot/levelSense/uplink.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      bits.substr(40, 8),
+      lifecycle.laserHWErrorFlag,
+    );
+    emit("sample", { data: ext, topic: "ext" });
+
+    data.laserReflectance = laserReflectance(bits.substr(48, 16));
+    data.temperature = temperature(bits.substr(64, 8));
+    data.tiltAngle = tiltAngle(bits.substr(72, 8), lifecycle.overTempFlag);
+    lifecycle.batteryVoltage = batteryVoltage(bits.substr(80, 8));
+    // Ultrasonic Transmission 8
```
Comment (mariusgiger) [inline types/pipiot/levelSense/uplink.js:195]: why is this comment here?

**Review by mariusgiger — APPROVED** [review]
please check the comments and find a better way for the cd variable

File: types/pipiot/peopleSense/uplink.js (hunk 1 above)
Comment (J2thatsme) [reply types/pipiot/peopleSense/uplink.js:36]: looks like i don't need it

File: types/pipiot/levelSense/uplink.js (hunk 2 above)
Comment (J2thatsme) [reply types/pipiot/levelSense/uplink.js:195]: Placeholder

File: types/pipiot/peopleSense/uplink.js (hunk 1 above)
Comment (mariusgiger) [reply types/pipiot/peopleSense/uplink.js:36]: it is still here? :D

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**